### PR TITLE
Move designer defaults from configration repo

### PR DESCRIPTION
**Pull request by syedimranhassan**

### CODE REVIEW

File: designer/settings/base.py [hunk 1]
```diff
@@ -117,7 +117,7 @@
 
 # MEDIA CONFIGURATION
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#media-root
-MEDIA_ROOT = root('media')
+MEDIA_ROOT = '/edx/var/designer/media'
```
Comment (bderusha) [inline designer/settings/base.py]: I think we may want to keep the original defaults here

File: designer/settings/base.py [hunk 2]
```diff
@@ -126,7 +126,7 @@
 
 # STATIC FILE CONFIGURATION
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#static-root
-STATIC_ROOT = root('assets')
+STATIC_ROOT = '/edx/var/designer/staticfiles'
```
Comment (bderusha) [inline designer/settings/base.py]: I think we may want to keep the original default here as well

File: designer/settings/base.py [hunk 3]
```diff
@@ -169,12 +169,12 @@
...
 SESSION_COOKIE_NAME = 'designer_sessionid'
 CSRF_COOKIE_NAME = 'designer_csrftoken'
-LANGUAGE_COOKIE_NAME = 'designer_language'
+LANGUAGE_COOKIE_NAME = 'openedx-language-preference'
 # END COOKIE CONFIGURATION
 
 # AUTHENTICATION CONFIGURATION
 LOGIN_URL = '/login/'
-LOGOUT_URL = '/logout/'
+LOGOUT_URL = 'http://localhost:18808/logout/'
```
Comment (bderusha) [inline designer/settings/base.py]: I'd love to know more about why we specify a port here.  Should we do it for login as well? Or is the relative path the better pattern?

**Comment by bderusha** [comment]
@georgebabey @tuchfarber We're movign defaults from configuration into the apps `base.py` file.  With the eventual goal of removing them from configuration entirely.  Can you weigh in on some of these defaults?

File: designer/settings/base.py (hunk 3 above)
Comment (tuchfarber) [reply designer/settings/base.py]: I do think this should probably just be the relative URL

File: designer/settings/base.py [hunk 4]
```diff
@@ -221,3 +221,12 @@
...
 WAGTAIL_SITE_NAME = 'Designer'
+
+CERTIFICATE_LANGUAGES= {
+    'en': 'English',
+    'es_419': 'Spanish'
+}
+DESIGNER_SERVICE_USER = 'designer_service_user'
+CSRF_COOKIE_SECURE = False
+SESSION_EXPIRE_AT_BROWSER_CLOSE = False
+SOCIAL_AUTH_REDIRECT_IS_HTTPS = False
```
Comment (coryleeio) [inline designer/settings/base.py]: Please be sure the base edx_django_service defaults get put in here too:


You need to include all these values as well:
https://github.com/edx/configuration/blob/master/playbooks/roles/edx_django_service/defaults/main.yml#L182-L222

as they get merged into the edx_django_service_config here:
https://github.com/edx/configuration/blob/master/playbooks/roles/edx_django_service/defaults/main.yml#L226

Comment (syedimranhassan) [reply designer/settings/base.py]: I have added all these varaiables and also I have moved databases to local.py and copy caches from local.py to base.py

**Pull request merged**